packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -74,10 +75,23 @@ export class DeltaStorageService implements IDeltaStorageService {
         to: number): Promise<IDeltasFetchResult> {
         const ordererRestWrapper = await RouterliciousOrdererRestWrapper.load(
             tenantId, id, this.tokenProvider, this.logger);
-        const ops = await ordererRestWrapper.get<ISequencedDocumentMessage[]>(this.url, { from, to });
+        const ops = await PerformanceEvent.timedExecAsync(
+            this.logger,
+            {
+                eventName: "getDeltas",
+                range: `${from}..${to}`,
```

Comment:
I'd suggest you to have separate fields - to & from.
Easier to do various analyzes with queries.

Reply:
Thanks for the tip! Will do